plugins/outputs/graylog/graylog.go

```diff
@@ -377,7 +377,7 @@ func (g *Graylog) connectRetry(tlsCfg *tls.Config) {
 
 	g.wg.Add(1)
 
-	unconnected := append([]string{}, g.Servers...)
+	unconnected := g.Servers
```

Comment:
Could we just use `g.Servers` directly or does it need to be a new variable?

Reply:
It must be `append` and with the current code, modifying `unconnected` will modify `g.Server` which **must** be avoided here!

Reply:
Fixed.